portal/common/lib/http/http_status_codes.ts

```diff
@@ -5,5 +5,6 @@
 export enum HttpStatusCodes {
     TOO_MANY_REDIRECTS = 310,
     NOT_FOUND = 404,
+    UNPROCESSABLE_ENTITY = 422,
```

Comment:
Nit: 422 is unprocessable content https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422

```suggestion
    UNPROCESSABLE_CONTENT = 422,
```